src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -148,9 +150,28 @@ public void resetTargetedHeight() {
     climbHeightTwo = m_climbMotorControllerTwo.getEncoder().getPosition();
   }
 
+  public void climbKeepDownFunction() {
+    m_climbMotorControllerOne
+        .getPIDCtrl()
+        .setReference(climbHeightOne, CANSparkMax.ControlType.kPosition);
+    sbclimbHeightOne.setNumber(climbHeightOne);
+
+    m_climbMotorControllerTwo
+        .getPIDCtrl()
+        .setReference(climbHeightTwo, CANSparkMax.ControlType.kPosition);
+    sbclimbHeightTwo.setNumber(climbHeightTwo);
+  }
+
   public void climbEnable() {
     climbEnabbled = !climbEnabbled;
     sbClimbEnabbled.setBoolean(climbEnabbled);
+    if (climbEnabbled) {
+      m_climbMotorControllerOne.setSmartCurrentLimit(60);
+      m_climbMotorControllerTwo.setSmartCurrentLimit(60);
+    } else {
+      m_climbMotorControllerOne.setSmartCurrentLimit(10);
+      m_climbMotorControllerTwo.setSmartCurrentLimit(10);
```

Comment:
It might be expensive to change the current limit mid-match. Do we need to do this?

Reply:
Yes, Andrew wants the have the climb run during the match and he wanted us to have a low current limit, but then we need to change it when we are actually climbing.